### PR TITLE
Resolve syncing conflicts from repo_sync_working_branch to live

### PR DESCRIPTION
**Pull request by openpublishbuild**
Pull request opened by Docs to resolve syncing conflicts from repo_sync_working_branch to live.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/communicate-with-users-from-other-organizations.md [hunk 1]
```diff
@@ -82,6 +82,8 @@ If you're ready to turn on guest access in your organization, start with the [Gu
 <sup>2</sup> Only by email or Session Initiation Protocol (SIP) address.<br>
 <sup>3</sup> External (federated) chat is 1:1 only.<br>
 <sup>4</sup> Supported for 1:1 chat for Teams Only to Teams Only users from two different organizations. 
+<sup>5</sup> By default, external participants can't see the phone numbers of dialed-in participants. If you want to maintain the privacy of these phone numbers, select **Tones** for **Entry/exit announcement type** (this prevents the numbers from being read out by Teams).If you don't want to reveal external PSTN phone numbers to external users, when you turn on **Entry/exit announcements**, select **Tones** for the announcement type. To learn more, read [Turn on or off entry and exit announcements for meetings in Microsoft Teams](turn-on-or-off-entry-and-exit-announcements-for-meetings-in-teams.md).
```
Comment (JohanFreelancer9) [inline Teams/communicate-with-users-from-other-organizations.md:85]:
```suggestion
<sup>5</sup> By default, external participants can't see the phone numbers of dialed-in participants. If you want to maintain the privacy of these phone numbers, select **Tones** for **Entry/exit announcement type** (this prevents the numbers from being read out by Teams). If you don't want to reveal external PSTN phone numbers to external users, when you turn on **Entry/exit announcements**, select **Tones** for the announcement type. To learn more, read [Turn on or off entry and exit announcements for meetings in Microsoft Teams](turn-on-or-off-entry-and-exit-announcements-for-meetings-in-teams.md).
```

File: Teams/manage-emergency-calling-policies.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ ms.custom: seo-marvel-apr2020
 
 # Manage emergency calling policies in Microsoft Teams
 
-If your organization uses Calling Plans or deployed Phone System Direct Routing, you can use emergency calling policies in Microsoft Teams to define what happens when a Teams user in your organization makes an emergency call. You can set who to notify and how they are notified when a user who is assigned the policy calls emergency services. For example, you can configure policy settings to automatically notify your organization's security desk and have them listen in emergency calls.  
+If your organization uses [Calling Plans](set-up-calling-plans.md) or deployed [Phone System Direct Routing](direct-routing-landing-page.md), you can use emergency calling policies in Microsoft Teams to define what happens when a Teams user in your organization makes an emergency call. You can set who to notify and how they are notified when a user who is assigned the policy calls emergency services. For example, you can configure policy settings to automatically notify your organization's security desk and have them listen in emergency calls.
```
Comment (JohanFreelancer9) [inline Teams/manage-emergency-calling-policies.md:26]:
```suggestion
If your organization uses [Calling Plans](set-up-calling-plans.md) or deployed [Phone System Direct Routing](direct-routing-landing-page.md), you can use emergency calling policies in Microsoft Teams to define what happens when a Teams user in your organization makes an emergency call. You can set who to notify and how they are notified when a user who is assigned the policy calls emergency services. For example, you can configure policy settings to automatically notify your organization's security desk and have them listen in on emergency calls.  
```

Comment (JohanFreelancer9) [reply Teams/manage-emergency-calling-policies.md:26]: Please change "listen in emergency calls" to "listen in on emergency calls" in the last sentence.

File: Teams/communicate-with-users-from-other-organizations.md (hunk 1 above)
Comment (JohanFreelancer9) [reply Teams/communicate-with-users-from-other-organizations.md:85]: Please add a space before "If".

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LanaChin** [comment]
Hi @Manuela - This is  a merge from the live branch that the PubOps team handles.  FYI @traya1.

**Pull request merged**